test/integration/edition_edit_test.rb

```diff
@@ -161,6 +196,73 @@ class EditionEditTest < IntegrationTest
     end
   end
 
+  context "Update important note page" do
+    should "render the 'Update important note' page" do
+      visit_draft_edition
+      click_link("History and notes")
+      click_link("Update important note")
+
+      within :css, ".gem-c-heading" do
+        assert page.has_css?("h1", text: "Update important note")
+        assert page.has_css?(".gem-c-heading__context", text: @draft_edition.title)
+      end
+
+      assert page.has_text?("Add important notes that anyone who works on this edition needs to see, eg “(Doesn’t) need fact check, don’t publish.”.")
+      assert page.has_text?("Each edition can have only one important note at a time.")
+
+      within :css, ".gem-c-textarea" do
+        assert page.has_css?("label", text: "Important note")
+        assert page.has_css?("textarea")
+      end
+
+      assert page.has_button?("Save")
+      assert page.has_link?("Cancel")
+    end
+
+    should "pre-populate with the existing note" do
+      note_text = "This is really really urgent!"
+      create_draft_edition
+      create_important_note_for_edition(@draft_edition, note_text)
+      visit edition_path(@draft_edition)
+
+      click_link("History and notes")
+      click_link("Update important note")
+
+      assert page.has_field?("Important note", with: note_text)
+    end
+
+    # TODO: Test to be switched on when the Edition notes history is implemented.
+    should "not show important notes in edition history" do
+      skip "Until this finctionality is complete: #603 - History and notes tab (excluding sidebar) [Edit page]"
+      note_text = "This is really really urgent!"
+      note_text_2 = "Another note"
+      note_text_3 = "Yet another note"
+      create_draft_edition
+      create_important_note_for_edition(@draft_edition, note_text)
+      create_important_note_for_edition(@draft_edition, note_text_2)
+      create_important_note_for_edition(@draft_edition, note_text_3)
+      visit edition_path(@draft_edition)
+      click_link("History and notes")
```

Comment:
I think this can be simplified
```suggestion
      visit history_edition_path(@draft_edition)
```